server/domains/project/entity.go

```diff
@@ -0,0 +1,73 @@
+package project
+
+import (
+	"database/sql"
+)
+
+// Project has a project model object
+type Project struct {
+	ID               int64
+	UserID           int64
+	Title            string
+	Description      string
+	RepositoryID     sql.NullInt64
+	ShowIssues       bool
+	ShowPullRequests bool
+	infrastructure   Repository
+}
+
+type Repository interface {
```

Comment:
__[Golint]__ exported type Repository should have comment or be unexported

<sub>posted by Sider</sub>